paper.md

```diff
@@ -75,7 +75,7 @@ F(q; k, \nu) = \frac{k\nu^{\nu/2}}{\Gamma(\nu/2)2^{\nu/2-1}}
 where $q$ is the studentized range, $k$ is the number of groups, $\nu$ is the
 number of degrees of freedom used to determine the pooled sample variance, and
 $\phi(z)$ and $\Phi(z)$ represent the normal probability density function
-and normal cumulative distribution function, respectively.
+and normal CDF, respectively.
```

Comment:
I debated about this and had chosen to spell it out because I spell out "probability density function" in the next sentence and (looking at this sentence alone) it looks strange to spell one out but not the other. Another option might be to just not add the acronym "CDF" above?

Reply:
I was thinking about this too as I was reading this section. It may be better not to add the "CDF" acronym and instead spell out "cumulative distribution function" everywhere in the paper.

Reply:
```suggestion
and normal cumulative distribution function, respectively.
```

Reply:
OK! I'll leave this alone and just change line 66.